sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/BlobDecryptionPolicy.java

```diff
@@ -431,12 +460,12 @@ private Cipher getCipher(byte[] contentEncryptionKey, EncryptionData encryptionD
 
     private Long blobSize(HttpHeaders headers) {
```

Comment:
Just a general nit on this whole method, the return can be changed to `long` and the calls to `Long.valueOf` can be changed to `Long.parseLong` as `Long.valueOf` is just a `new Long(Long.parseLong)` wrapper and will throw an exception if the String value is null.

Or, the other suggestion is keeping this `Long` and null checking the inputs to `Long.valueOf`